File: ML/Ayushsingh07_Ayushkumarsingh_2024cse1037_2/gettingstarted/primefactors.py
import math
n=int(input())
while n%2==0:
    print(2,end=" ")
    n=n/2
for i in range(3,int(math.sqrt(n))+1,2):
    while (n % i == 0):
         print (int(i),end=" ")
         n = n / i
    
if n>2:
    print (int(n),end=" ")

File: AR-VR/tanya007_Tanya_2024ec1197_2/gettingStarted/PrimeFactors.cpp
#include<iostream>
#include<math.h>
using namespace std;

int main(){
    int n;
    cin >> n;
    
    for(int i=2;i<sqrt(n);i++){

        while(n%i == 0){
            cout<<i<<" ";
            n/=i;
        }
    }
    if(n>2)
        cout<<n;
}